src/main/java/lotto/LottoMaking.java

```diff
@@ -0,0 +1,8 @@
+package lotto;
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class LottoMaking {
```

Comment:
생성의 책임을 지닌 객체를 보통 Factory로 많이 명명해요!
```suggestion
public class LottoFactory {
```